src/language/functions.md

```diff
@@ -500,6 +500,25 @@ Iterable<int> naturalsDownFrom(int n) sync* {
 }
 ```
 
+## External functions {#external}
+
+An external function is a function whose body is provided separately from its declaration.
```

Comment:
Starting with this does seem safer, thanks! However, do consider briefly introducing the following code snippet/syntax, otherwise it's not clear why it's there/what it means.